openvdb_houdini/SOP_OpenVDB_Activate.cc

```diff
@@ -0,0 +1,604 @@
+// Copyright Contributors to the OpenVDB Project
+// SPDX-License-Identifier: MPL-2.0
+//
+/// @file SOP_OpenVDB_Activate.cc
+///
+/// @author FX R&D OpenVDB team
+///
+/// @brief Activate VDBs according to various rules
+
+#if 1
```

Comment:
I tried that, but we tend to define SESI_OPENVDB farther in the build process (I don't think it is on the command line -D options) so I couldn't use that there.

The issue is our includes need slightly different paths, which is an annoying difference on all the current nodes.  I wasn't sure if I'd send this with only the OpenVDB includes, but figured maybe a #if makes the difference smaller and more manageable.

Reply:
Ok, no problem. Without making bigger changes for now, a quick comment about what it's for would be helpful?

Longer-term, I think we should re-open the discussion as to whether we should merge houdini_utils and openvdb_houdini libraries to simplify our codebase. Now that OP_NodeChain is deprecated, ParmFactory classes are the only reason to have this distinction and since the SESI recommended approach has changed to DialogScript, it doesn't feel as mission-critical to be shipping a Houdini utility library to make SOP UI building easier.

Reply:
That is something for DreamWorks to weight in on, it might really adversely affect their merging if we combined them.  Personally I'm all for it :>